.github/workflows/release.yml

```diff
@@ -70,8 +87,10 @@ jobs:
           # Avoid persisting GITHUB_TOKEN credentials as they take priority over our service account PAT for `git push` operations
           # More details: https://github.com/actions/checkout/blob/b4626ce19ce1106186ddf9bb20e706842f11a7c3/adrs/0153-checkout-v2.md#persist-credentials
           persist-credentials: false
-      - name: Update meta package SDKVersion
-        run: sed -i "s/var SDKVersion =.*/var SDKVersion = \"${{ needs.changelog-version.outputs.version }}\"/" meta/meta.go
+      - name: Update meta package SDKVersion and SDKPrerelease
+        run: |
+          sed -i "s/var SDKVersion =.*/var SDKVersion = \"${{ needs.changelog-version.outputs.version_only }}\"/" meta/meta.go
+          sed -i "s/var SDKPrerelease =.*/var SDKPrerelease = \"${{ needs.changelog-version.outputs.prerelease }}\"/" meta/meta.go
```

Comment:
Honestly.... I'm wondering if we should just include the full version in `SDKVersion` and then the `SDKPrerelease` as the separated portion..

Because the `SDKVersion` shouldn't be `v2.37.0` when we release this, it should really be `v2.37.0-alpha.1`. I don't think `SDKPrerelease` is super useful given that context, but since it exists, probably best to keep it up to date 😆 

So when we release I think it'd make sense to be:
```go
var SDKVersion = "v2.37.0-alpha.1"
var SDKPrerelease = "alpha.1"
```

Rather than:
```go
var SDKVersion = "v2.37.0"
var SDKPrerelease = "alpha.1"
```

Reply:
I guess also, all of this is deprecated anyways, so it doesn't really matter 🤷🏻

Reply:
I'm okay with prioritizing `SDKVersionString()` correctness. This is consistent with the doc comment for `SDKVersion`: `The main version number that is being run at the moment.`